src/components/organisms/curriculum/OakDownloadsJourneyChildSubjectTierSelector/OakDownloadsJourneyChildSubjectTierSelector.tsx

```diff
@@ -0,0 +1,172 @@
+import React, { useState } from "react";
+import styled, { css } from "styled-components";
+
+import { OakBox, OakHeading } from "../../../atoms";
+
+// import { parseSpacing } from "@/styles/helpers/parseSpacing";
+import {
+  OakPrimaryButton,
+  OakRadioButton,
+  OakRadioGroup,
+} from "@/components/molecules";
+
+// const StyledOakFlex = styled(OakFlex)`
+//   width: ${parseSpacing("all-spacing-6")};
+// `;
+// For example you could restyle the OakFlex component by adding the styles to the css template literal below
+export interface Tier {
+  tier: string;
+  tier_slug: string;
+}
+export interface Subject {
+  subject: string;
+  subject_slug: string;
+}
+
+export type OakDownloadsJourneyChildSubjectTierSelectorProps = {
+  /* An array of Tier objects containing `tier` and `tier_slug` */
+  tiers: Tier[];
+  /* An array of Subject objects containing `subject` and `subject_slug` */
+  childSubjects?: Subject[];
+  /* Callback function which returns the selected tier and subject once the Next button is pressed. */
+  getTierSubjectValues: (
+    tierSlug: string,
+    childSubjectSlug: string | null,
+  ) => void;
+};
+
+// By adding the style css utils to this components css your component will be able to accept corresponding props and prop values.
+// you can also add custom styles to the component by adding the styles to the css template literal below
+
+const OakDownloadsJourneyChildSubjectTierSelectorCss = css<OakDownloadsJourneyChildSubjectTierSelectorProps>``;
+
+/**
+ *
+ * add default and custom styles to the component by adding the styles to the css template literal below
+ *
+ * ${typographyStyle}
+ * ${colorStyle}
+ * ${spacingStyle}
+ * ${displayStyle}
+ * ${borderStyle}
+ * ${dropShadowStyle}
+ * ${colorFilterStyle}
+ *
+ */
+
+const UnstyledComponent = (
+  props: OakDownloadsJourneyChildSubjectTierSelectorProps,
+) => {
+  const { childSubjects, tiers, getTierSubjectValues } = props;
+
+  const [childSubjectSelected, setChildSubjectSelected] = useState<
+    string | null
+  >(childSubjects && childSubjects[0] ? childSubjects[0]?.subject_slug : null);
+  const [tierSelected, setTierSelected] = useState<string>("foundation");
+
+  function handleChildSubjectSelection(
+    e: React.ChangeEvent<HTMLInputElement>,
+  ): void {
+    const childSubjectSlug = e.currentTarget.value;
+    setChildSubjectSelected(childSubjectSlug);
+  }
+
+  function handleTierSelection(e: React.ChangeEvent<HTMLInputElement>): void {
+    const tierSlug = e.currentTarget.value;
+    setTierSelected(tierSlug);
+  }
+
+  function handleNextClick() {
+    getTierSubjectValues(tierSelected, childSubjectSelected);
+  }
+
+  return (
+    <OakBox>
+      <OakHeading $font={"heading-4"} tag="h4" $mb={"space-between-m"}>
```

Comment:
I'm personally anti using margin whenever possible. It breaks component encapsulation in a subtle way and leads to layout issues down the line. A combination of Flexbox gap and padding normally does the same job.